src/features/game/types/pets.ts

```diff
@@ -50,6 +53,31 @@ export type PetCategory =
   | "Hamster"
   | "Penguin";
 
+export type PetCategoryName =
+  | "Guardian"
+  | "Hunter"
+  | "Voyager"
+  | "Beast"
+  | "Moonkin"
+  | "Snowkin"
+  | "Forager";
+
+export type Pet = {
+  name: PetName | number;
+  requests: {
+    food: CookableName[];
+    foodFed?: CookableName[];
+    fedAt?: number;
+  };
+  energy: number;
+  experience: number;
+};
+
+export type Pets = {
+  common: Partial<Record<PetName, Pet>>;
```

Comment:
Can `common` be undefined?

My PetModal and PetHouseModal were throwing errors since I had old pets placed. I had to nuke my DB to get it working again since `common` was not yet defined for me.

Consider keeping this as optional to avoid bricking players who have pets placed

<img width="585" height="185" alt="Screenshot 2025-09-03 at 7 36 43 AM" src="https://github.com/user-attachments/assets/ac038594-53b6-479a-81cb-62b4c5c40375" />